### PR TITLE
[BEAM-1652] Code style instructions in contribution guide

### PR DESCRIPTION
**Pull request by aviemzur**

### CODE REVIEW

**Comment by aviemzur** [comment]
R: @davorbonaci

File: src/contribute/contribution-guide.md [hunk 1]
```diff
@@ -143,11 +143,24 @@ clicking the "Check Module" button. The scan should report no errors.
 Note: Selecting "Check Project" may report some errors from the archetype
 modules as they are not configured for Checkstyle validation.
 
+###### Code Style
+Intellij supports code styles within the IDE. Use one of the following to ensure your code style
```
Comment (davorbonaci) [inline src/contribute/contribution-guide.md]: casing: IntelliJ

File: src/contribute/contribution-guide.md [hunk 2]
```diff
@@ -143,11 +143,24 @@ clicking the "Check Module" button. The scan should report no errors.
...
+matches the project's checkstyle enforcements.
+
+1. (Option 1) Configure IntelliJ to use beam-codestyle.xml
```
Comment (davorbonaci) [inline src/contribute/contribution-guide.md]: Can you be consistent in adding '.' (dot) at the end of each bullet -- I think that's what most other lists currently do.

**Comment by aviemzur** [comment]
@davorbonaci made the fixes you requested.

**Pull request closed without merging**